db/migrate/20230418075940_assing_fqdn_to_host_name.rb

```diff
@@ -0,0 +1,11 @@
+class AssingFqdnToHostName < ActiveRecord::Migration[6.1]
+  def up
+    Host.all.each do |host|
```

Comment:
how long this will take on a system with e.g. 10k hosts? `Host.all` is always tricky, should we do this `in_batches`?

Reply:
I am not sure how long will it take on a 10k host machine, that might be interesting to test. 
I have added `find_in_batches` with batch size of 1000, which should be fine. But if you have a suggestion on a badge size, let me know.

Reply:
I have just tested the migration with 30k hosts. The result:
`== 20230418075940 AssignFqdnToHostName: migrated (72.1858s) ===================`

Reply:
Ok, that does not sound too bad